src/allDrugs.jl

```diff
@@ -120,7 +146,7 @@ function optim_all(concs::Array{Float64, 2}, g1::Array{Float64, 3}, g2::Array{Fl
         costCenter = f(hillParams)
 
         # Handle the integer-valued parameters
-        for ii = 34:37
+        for ii = 33:37
```

Comment:
E.g., if the problem is predict(), why do you need this change?

Reply:
Sorry, this was just my mistake. I change a lot of things and when bringing back the code I must have made a mistake. This shouldn't change at all. I will correct it right away.